Bottom: a66edab552214e8dd9c35a42955ca638fd3f2067
Top:    7b9dc9a8eddeff5ad64c7555e8cf024c0491e40d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 18:42:45 +0000

Refresh of board-outline-polygon-generati

---

diff --git a/src/polygon.c b/src/polygon.c
index e284458..02c3d04 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -2099,8 +2099,8 @@ POLYAREA *board_outline_poly (void)
   /* Now we just need to work out which pieces of polygon are inside
      and outside the board! */
 
-  /* If there is only one piece, return that */
-  if (clipped->f == clipped)
+  /* If there is no result, or only one piece, return that */
+  if (clipped == NULL || clipped->f == clipped)
     return clipped;
 
   /* WARNING: This next check is O(n^2), where n is the number of clipped
